HelpfulItem.h

```diff
@@ -10,6 +10,6 @@
 struct HelpfulItem : Item
```

Comment:
Don't include what you don't use.
Take a look at the previous lines and determine if you need to include any of these header files.
- [ ] revise